docs/minio-admin-complete-guide.md

```diff
@@ -203,7 +203,7 @@ This option disables the color theme. It is useful for dumb terminals.
 ### Option [--quiet]
 Quiet option suppress chatty console output.
 
-### Option [--config-folder]
+### Option [--config-directory]
```

Comment:
can we name this --config-dir instead of --config-directory? minio server uses --config-dir as well, naming will be consistent

Reply:
Done